documentation/style-guide.rst

```diff
@@ -124,11 +124,37 @@ explanation.
   provide context, make connections between topics, and discuss alternative
   opinions. There is no section dedicated to explanations but these can be
   found throughout Python's documentation, for example the
-  :ref:`python:unicode-howto`
+  :ref:`python:unicode-howto`.
 
 Please consult the `Diátaxis <https://diataxis.fr/>`_ guide for more
 detail.
 
+Links
+=====
+
+Linking words to more information about those words is a powerful tool for
+helping people navigate documentation, but links can be over-used.
+
+Links should be provided only if they help the user.  If a link will not help
+the user, do not create a link.  You may need to suppress a link that Sphinx
+would have created automatically.
+
+Generally, a link should be provided for the first use of a term in a unit This
+is not a hard and fast rule.  Sometimes the second mention is a more
+appropriate jumping off point.  Some units are long enough to have a few
```

Comment:
```suggestion
appropriate for a link.  Some units are long enough to have a few
```

Reply:
@nedbat I'm not sure that "jumping off point" is commonly used globally.

Reply:
Done.